specificatie/datum.yaml

```diff
@@ -40,12 +40,16 @@ components:
             datum:
               type: string
               format: date
+            langFormaat:
```

Comment:
toevoegen van een pattern? voorstel: ^[a-z0-9 ]+$

Reply:
> toevoegen van een pattern? voorstel: ^[a-z0-9 ]+$

Of nog scherper:

`^(1[0-9]|(2)([0-9]{1})|(3)([0-1]{1})|([1-9]{1})) (januari|februari|maart|april|mei|juni|juli|augustus|september|oktober|november|december) ([1-2]{1})([0-9]{3})$`

waarbij hier (nog) geen rekening wordt gehouden met het feit dat niet alle maanden 31 dagen hebben.